README.rst

```diff
@@ -1,4 +1,4 @@
-Cekit
+CEkit
```

Comment:
It should be: `CEKit`

Reply:
Darn, thought I caught all of those! ;-)